plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -113,30 +112,32 @@ library PaymentProcessInFlightExit {
         }
     }
 
-    function isAnyInputSpent(
+    // input is unspent for a given exitId if it's not marked in finalized outputIds map
+    // or it's marked with the same exitId
```

Comment:
First line is rather obvious - follows from the name of the function. Second line's message is very not obvious - question pops up immediately: "why would it ever be marked by the same exitId?"

This is "what" comment, not "why" comment.